hibernate-core/src/main/java/org/hibernate/graph/EntityGraphParser.java

```diff
@@ -0,0 +1,257 @@
+/*
+ * Hibernate, Relational Persistence for Idiomatic Java
+ *
+ * License: GNU Lesser General Public License (LGPL), version 2.1 or later.
+ * See the lgpl.txt file in the root directory or <http://www.gnu.org/licenses/lgpl-2.1.html>.
+ */
+package org.hibernate.graph;
+
+import javax.persistence.EntityManager;
+
+import org.hibernate.Session;
+import org.hibernate.engine.spi.SessionFactoryImplementor;
+import org.hibernate.engine.spi.SessionImplementor;
+import org.hibernate.graph.internal.RootGraphImpl;
+import org.hibernate.graph.internal.parse.GraphParser;
+import org.hibernate.graph.spi.GraphImplementor;
+import org.hibernate.graph.spi.RootGraphImplementor;
+
+/**
+ * todo (graph) : to me most of this seems more appropriate in documentation as opposed to here
```

Comment:
Is it still the case? In the documentation, you point to this exact class to get more information.

Reply:
Well, as a user, would you rather:

1. Read the user guide and understand how to write these "entity graph language" strings?
2. Read the user guide and and be directed to instead go read some (not even linked) Javadocs to understand how to write these "entity graph language" strings?

Reply:
I would prefer having the information in the documentation. It's just that it was not clear the patch was a preview and you wanted to do that before the final inclusion so my remark was more: either we fix it or we remove the todo as it's a bit unintuitive to point the user there and start with a todo saying we should include it in the doc :).

Reply:
Sure, but that's kind of the point of todos ;)